sim/druid/swipe.go

```diff
@@ -62,18 +50,31 @@ func (druid *Druid) registerSwipeBearSpell() {
 			IgnoreHaste: true,
 		},
 
-		DamageMultiplier: 1 + 0.1*float64(druid.Talents.SavageFury),
+		DamageMultiplier: baseMultiplier + 0.1*float64(druid.Talents.SavageFury),
```

Comment:
A base damage multiplier should be applied to the `BaseDamageMultiplierAdditive` field. It looks like Savage Fury is an additive full damage multiplier so it should technically be applied to `DamageMultiplierAddtitive` instead, too